### PR TITLE
Contact me form

### PR DESCRIPTION
**Pull request by MattGomb**
Created the last part of the mobile version by:

- adding a new grid element that is the last 'page', the contact-me form
- adding the background
- adding the form
- styling the form
- styling the whole element to scale and fit into the project

### CODE REVIEW

**Comment by bobb-Rob** [comment]
@MattGomb Please fix the linter issues on the style file

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @MattGomb , 

Good job so far! 👏👏👏 
You did excellently well implementing the requirements in this project, but there are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MattGomb** [comment]
@bobb-Rob  hi, thank you for the suggestions, they are fixed now! :)

File: styles.css [hunk 1]
```diff
@@ -392,3 +393,93 @@ span {
...
+  padding-top: 103px;
+}
+
+.p-contact-me {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: justify;
```
Comment (franclobo) [inline styles.css:428]: - [ ] **[OPTIONAL]** Kindly use `text-align: center` to stick to the design as much as possible. 📑 

text-center

File: styles.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
+* {
+  outline: auto;
```
Comment (franclobo) [inline styles.css:2]: - [ ] **[OPTIONAL]** Please, 🙏🏽 we could avoid using `outline: auto` to delete the border around elements.

- With outline

With-outline

- Without outline

Without-outline

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @MattGomb , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- No linter errors ✔️ 
- Correct Github Flow 💯 
- Descriptive PR ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MattGomb** [comment]
@franclobo thank you for the suggestions, fixed :)

**Review by Reem-lab — APPROVED** [review]
# Hi @MattGomb, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the design✅
📌 Good commit messages ✅
📌Professional README file. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**